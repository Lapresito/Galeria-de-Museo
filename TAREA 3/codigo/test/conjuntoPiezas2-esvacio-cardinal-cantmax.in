crearConjuntoPiezas 2
esVacioConjuntoPiezas
cardinalConjuntoPiezas
cantMaxConjuntoPiezas
insertarConjuntoPiezas 2
cardinalConjuntoPiezas
perteneceConjuntoPiezas 4
cardinalConjuntoPiezas
perteneceConjuntoPiezas 10
esVacioConjuntoPiezas
cardinalConjuntoPiezas
cantMaxConjuntoPiezas
liberarConjuntoPiezas
Fin