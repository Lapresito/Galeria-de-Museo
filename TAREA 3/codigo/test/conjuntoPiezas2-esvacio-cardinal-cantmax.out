Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El conjunto piezas fue creado en forma correcta.
2> El conjunto de piezas es vacio.
3> El cardinal del conjunto es: 0.
4> La cantidad maxima del conjunto es: 2.
5> Se ejecuto 'insertarTConjuntoPiezas' con pieza de id 2 exitosamente.
6> El cardinal del conjunto es: 0.
7> La pieza de id 4 NO pertenece al conjunto.
8> El cardinal del conjunto es: 0.
9> La pieza de id 10 NO pertenece al conjunto.
10> El conjunto de piezas es vacio.
11> El cardinal del conjunto es: 0.
12> La cantidad maxima del conjunto es: 2.
13> Conjunto de piezas liberado exitosamente.
14> Fin.
